dpnp/dpnp_iface_mathematical.py

```diff
@@ -3064,6 +3065,67 @@ def prod(
 )
 
 
+def real_if_close(a, tol=100):
+    """
+    If input is complex with all imaginary parts close to zero, return real
+    parts.
+
+    "Close to zero" is defined as `tol` * (machine epsilon of the type for `a`).
+
+    For full documentation refer to :obj:`numpy.real_if_close`.
+
+    Parameters
+    ----------
+    a : {dpnp.ndarray, usm_ndarray}
+        Input array.
+    tol : scalar, optional
+        Tolerance in machine epsilons for the complex part of the elements in
+        the array. If the tolerance is <=1, then the absolute tolerance is used.
+        Default: ``100``.
+
+    Returns
+    -------
+    out : dpnp.ndarray
+        If `a` is real, the type of `a` is used for the output. If `a` has
+        complex elements, the returned type is float.
+
+    See Also
+    --------
+    real, imag, angle
```

Comment:
Could you expand the description of these functions to follow the general style?
Like this:
```
:obj:`dpnp.real` : The real part of the array.
...
```

Reply:
Done. Thank you for noticing that.